packages/phrases/src/locales/zh-cn/translation/admin-console/connector-details.ts

```diff
@@ -37,9 +37,9 @@ const connector_details = {
       '内置电子邮件使用默认模板，无需配置即可无缝发送验证电子邮件。您可以自定义基本品牌信息。',
     description_action_text: '查看模板',
     from_email_field: '发件人电子邮件',
-    from_name_field: '发件人姓名',
-    from_name_tip: '自定义电子邮件的发件人姓名。如果留空，则默认使用“Verification”作为名称。',
-    from_name_placeholder: '发件人姓名',
+    sender_name_field: '发件人姓名',
+    sender_name_tip: '自定义电子邮件的发件人姓名。如果留空，则默认使用“Verification”作为名称。',
```

Comment:
The `Verification` should also be translated?

Reply:
`Verification` is the subject of the email, we do not support subject/content i18n for email templates for now.